plugin/src/main/java/org/jenkinsci/plugins/structs/SymbolLookup.java

```diff
@@ -180,15 +181,17 @@ public static SymbolLookup get() {
      * @param c A class.
      * @return The {@link Symbol} annotation value(s) for the given class, or an empty {@link Set} if the annotation is not present.
      */
+    @SuppressFBWarnings(value = "NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE") // Because Findbugs wants to be difficult.
```

Comment:
Use `justification` and explain what the null return value is. Maybe something you should actually defend against.